Final_Project_Paper-AR.qmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Final_Project_Paper"
+Author: "Jeffrey Gilbert"
+format: html
+---
+
+The expansion of imperial powers in the Western Hemisphere has been a foreign policy concern for the United States, which was exemplified by the Monroe Doctrine as well as the Corollary to the Monroe Doctrine.^[fn1: Jeffrey A. Engel, et al., America in the World: A History in Documents from the War with Spain to the War on Terror, (Princeton, NJ: Princeton University Press, 2014), 23-24, 71-72] The fear of foreign influence in Latin America was especially prevalent during the interwar period and World War II. Franklin Delano Roosevelt was keenly aware of Latin America’s distaste for U.S. military intervention, so he decided to deviate from past precedence by taking the approach of being a good neighbor.^[fn2: Irwin F. Gellman, Good Neighbor Diplomacy: United States Policies in Latin America, 1933-1945, (Baltimore: Johns Hopkins University Press, 2019),11-12] Drawing from the digital methods used on the Foreign Policy Bulletin Dataset, which is comprised of publications from the Foreign Policy Bulletin, the interactions driving U.S. foreign policy during Roosevelt’s presidency become more apparent. Even though Latin America’s coverage was underrepresented compared to European or Axis nations, the Foreign Policy Bulletin’s coverage of Latin America suggests that collective security efforts and hemispheric solidarity were central to U.S. foreign policy efforts. Inter-American cooperation and hemispheric solidarity were seen as essential foreign policy measures for protecting the United States from the Axis Powers. 
```

Comment:
See my email for feedback on this paragraph. I think an effective opening hook here might be FDR's address that he delivered _to_ FPA members.